src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -146,6 +146,9 @@
             <tr>
                 <td class="font-normal">
                     <span class="label status-{{ symfony_status_class[collector.symfonystate] }}">{{ symfony_status[collector.symfonystate]|upper }}</span>
+                    {% if collector.symfonylts %}
```

Comment:
should be 
```suggestion
                    {% if collector.symfonyLts %}
```
IMO, as the corresponding method is named `isSymfonyLts`.
Even if php is not case sensitive on methods calls, we should use the right name

Reply:
hi @kevin-verschaeve 
thank for the comment
can you have a look at the upper comments flow? https://github.com/symfony/symfony/pull/31539#discussion_r286017148
I explained why I have written this way :)

Reply:
Oops, I need to read closed comments :smile: